src/tools/scripts/mattermost/createMissingAccount.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../../utils');
+const { loginAPI, searchUser, createAccount } = require('../../../utils/mattermost');
+const slugify = require('slugify');
+const { v4: uuidv4 } = require('uuid');
+
+// On crée les comptes pour les 21 conseillers en erreurs
+// Ils pourront utiliser la fonction "mot de passe oublié" pour se changer leur mot de passe et se connecter
+execute(__filename, async ({ app, db, logger, Sentry }) => {
+  const mattermost = app.get('mattermost');
+  const token = await loginAPI({ mattermost });
+
+  const sleep = ms => new Promise(r => setTimeout(r, ms));
+  let count = 0;
+  const conseillers = await db.collection('conseillers').find({
+    'statut': { $ne: 'RUPTURE' },
+    'mattermost.error': { $eq: true }
+  }).toArray();
+
+  for (const conseiller of conseillers) {
+    try {
+      const result = await searchUser(mattermost, token, conseiller);
+      if (result.data.length === 0) {
+        const nom = slugify(`${conseiller.nom}`, { replacement: '-', lower: true, strict: true });
+        const prenom = slugify(`${conseiller.prenom}`, { replacement: '-', lower: true, strict: true });
+
+        const login = `${prenom}.${nom}`;
+        const gandi = app.get('gandi');
+        const email = `${login}@${gandi.domain}`;
+        const password = uuidv4() + 'AZEdsf;+:'; // pour respecter la règle de complexité de mot de passe
+
+        createAccount({ mattermost, conseiller, email, login, password, db, logger, Sentry });
+
+        count++;
+      }
+    } catch (e) {
+      Sentry.captureException(e);
+      logger.error(e);
+    }
+
```

Comment:
Est-ce qu'in ne manque pas une façon de changer `mattermost.error` à false ? De même comme il y a la création du compte Gandi, est-ce qu'il ne faut pag gérer le emailCnError ?

Reply:
non car l'erreur peut être d'une autre nature. Il peut très bien y avoir une erreur et pour autant que le compte soit créé. il reste quelques cas à investiguer (sans impact sur la réussite de la création du compte; c'est en lien avec les canaux mattermost).

je ne modifie pas les emails dans cette PR, ce sera dans une autre (pas de création de boite email ici)